docs/developer/decision-records/2022-06-20-mvd-onboarding/README.md

```diff
@@ -0,0 +1,35 @@
+# Participants onboarding in MVD
+
+This document outlines the infrastructure steps for the scenario of onboarding participants to the Minimum Viable Dataspace (MVD).
+
+## Precondition - GAIA-X membership
+
+To join the MVD, each participant must have a Verifiable Credential (VC) signed by GAIA-X Authority which proves GAIA-X membership.
+
+At present the MVD deployment pipeline does not interact with the real GAIA-X Authority. A simulated GAIA-X Authority is deployed within MVD for demonstration purposes.
```

Comment:
Just a question: Do we plan to create a new separate service to simulate GAIA-X Authority?

Reply:
No, as far as I know we plan to only deploy DID document for GAIA-X Authority.

So I think this sentence in ADR is confusing, because we won't deploy the whole Gaia-X Authority service but only DID web. @algattik?

Reply:
Correct @Izzzu

Reply:
I changed the  `A simulated GAIA-X Authority` to `A simulated GAIA-X Authority DID document` to make it clear that we don't deploy any services for GAIA-X